src/firefly/js/templates/fireflyviewer/LandingPage.jsx

```diff
@@ -99,6 +101,11 @@ function EmptyResults({icon, text, subtext, actionItems}) {
                         </React.Fragment>
                     ))}
                 </Stack>
+                {Boolean(summaryText) &&
+                    <Stack spacing={2} alignItems='center'>
+                        <Typography level={'h3'} fontWeight={'var(--joy-fontWeight-md)'}>{summaryText}</Typography>
```

Comment:
Because this is at the bottom - styling and layout of it is kinda looking off to me compared to rest of components. I don't know how to fix it yet - I can play around with it if you want?

Reply:
I agree. I am against even having this here. If you want to try and idea would like to play with it then just do it in this branch.